packages/machine/src/models/state-channel.ts

```diff
@@ -304,7 +304,8 @@ export class StateChannel {
       this.userNeuteredExtendedKeys,
       appInstances,
       this.ethVirtualAppAgreementInstances,
-      this.freeBalanceAppIndexes
+      this.freeBalanceAppIndexes,
+      this.monotonicNumInstalledApps + 1
```

Comment:
Don't think we need this + 1

Reply:
why not? we need the number to increment for the next time someone calls this function

Reply:
Ah, my bad, yeah